packages/bsky/src/feature-gates.ts

```diff
@@ -0,0 +1,67 @@
+import { Statsig, StatsigUser } from 'statsig-node'
+import { sha256Hex } from '@atproto/crypto'
+
+export type Config = {
+  apiKey?: string
+  env?: 'development' | 'staging' | 'production' | string
+}
+
+export enum GateID {
+  NewSuggestedFollowsByActor = 'new_sugg_foll_by_actor',
+}
+
+/**
+ * @see https://docs.statsig.com/server/nodejsServerSDK
+ */
+export class FeatureGates {
+  ready = false
+  private statsig = Statsig
+  ids = GateID
+
+  constructor(private config: Config) {}
+
+  async start() {
+    try {
+      if (this.config.apiKey) {
+        await this.statsig.initialize(this.config.apiKey, {
+          environment: {
+            tier: this.config.env || 'development',
+          },
+        })
+        this.ready = true
+      } else if (process.env.NODE_ENV === 'test') {
```

Comment:
Would it be possible to hoist this behavior up either to `config.ts` or even the appview's statsig config in the tests?  This isn't a major deal / can totally punt for now and just leave a note if it's a pain!  The reason we've been trying to do things that way is just to avoid behaviors that don't flow from the app entrypoint/config, and perhaps a sprinkle of brain being a little wired to certain [12 factor](https://arc.net/l/quote/yrqrzool) thinking 🤣 .